app/packages/core/src/Root/Root.tsx

```diff
@@ -53,6 +53,29 @@ const rootQuery = graphql`
   }
 `;
 
+export const DatasetSavedViewsQuery = graphql`
+  query RootDatasetSavedViewsQuery($name: String!) {
+    ...RootDatasetSavedViewsFragment
+  }
+`;
+
+export const DatasetSavedViewsFragment = graphql`
+  fragment RootDatasetSavedViewsFragment on Query
+  @refetchable(queryName: "EootDatasetSavedViewsQuery") {
```

Comment:
@manivoxel51 typo?
Can this be changed to `RootDatasetSavedViewsQuery` and get rid of `export const DatasetSavedViewsQuery...`? You could then use RootDatasetSavedViewsQuery$key or something along those lines in the places where you call `useRefetchableFragment`

Reply:
agreed we should fix the typo - fixed!
agreed that we could consolidate the query - can tackle that later though